projects/1.18/assets/art-of-forging-a-tetra-addon/art_of_forging/lang/zh_cn.json

```
@@ -0,0 +1,431 @@
+	"tetra.advancements.scrolls.find_scroll_mace.description": "找到记载着钉锤的制作方法的卷轴",
+
+	"tetra.advancements.scrolls.root.title": "秘密（§eAoF§r）",
+	"tetra.advancements.scrolls.root.description": "找到锻造之艺的各工艺图纸",
+
+	"tetra/schematic/sword/bastard_blade.name": "变种剑剑刃",
+	"tetra/schematic/sword/bastard_blade.description": "将当前剑刃替换为变种剑剑刃，一种实用的剑刃\n \n材料§f硬度§r增加§f伤害§r",
+	"tetra/schematic/sword/bastard_blade.slot1": "剑刃材料",
+
+	"tetra.module.sword/bastard_blade.item_name": "变种剑",
+	"tetra.module.sword/bastard_blade.name": "变种剑剑刃",
+	"tetra.module.sword/bastard_blade.description": "一种实用的剑刃",
+	"tetra.module.sword/bastard_blade.material_name": "%s变种剑剑刃",
+	"tetra.module.sword/bastard_blade.prefix": "%s",
+
+	"tetra.advancements.upgrades.bastard_upgrade.title": "变种锻造",
+	"tetra.advancements.upgrades.bastard_upgrade.description": "制作一个变种剑",
+
+	"tetra/schematic/sword/executioner_blade.name": "行刑剑剑刃",
+	"tetra/schematic/sword/executioner_blade.description": "将当前剑刃替换为行刑剑剑刃，一种可怖的剑刃\n \n材料§f硬度§r增加§f伤害§r",
+	"tetra/schematic/sword/executioner_blade.slot1": "剑刃材料",
+
+	"tetra.module.sword/executioner_blade.item_name": "行刑剑",
+	"tetra.module.sword/executioner_blade.name": "行刑剑剑刃",
+	"tetra.module.sword/executioner_blade.description": "一种可怖的剑刃",
+	"tetra.module.sword/executioner_blade.material_name": "%s行刑剑剑刃",
+	"tetra.module.sword/executioner_blade.prefix": "%s",
+
+	"tetra.advancements.upgrades.executioner_upgrade.title": "砍下他们的头！",
+	"tetra.advancements.upgrades.executioner_upgrade.description": "制作一柄行刑剑",
+
+	"tetra.advancements.scrolls.craft_scroll_flamberge.title": "你能忍受这热度吗？",
+	"tetra.advancements.scrolls.craft_scroll_flamberge.description": "合成一个焰形剑卷轴",
+
+	"tetra.advancements.scrolls.craft_scroll_crucible.title": "地狱远征",
+	"tetra.advancements.scrolls.craft_scroll_crucible.description": "合成传说级的裁决剑卷轴",
+
+	"tetra/schematic/sword/rending_scissor_red.name": "§4片剪太刀[红]§r",
+	"tetra/schematic/sword/rending_scissor_red.description": "用硬化的生命纤维制成的剪刀形剑刃",
+	"tetra/schematic/sword/rending_scissor_red.slot1": "片剪太刀",
+
+	"tetra.module.sword/rending_scissor_red.item_name": "片剪太刀",
+	"tetra.module.sword/rending_scissor_red.name": "片剪太刀",
+	"tetra.module.sword/rending_scissor_red.description": "用硬化的生命纤维制成的剪刀形剑刃",
+	"tetra.variant.rending_scissor_red/rending_scissor_red": "§4片剪太刀[红]§r",
+
+	"tetra/schematic/sword/rending_scissor_purple.name": "§5片剪太刀[紫]§r",
+	"tetra/schematic/sword/rending_scissor_purple.description": "用硬化的生命纤维制成的剪刀形剑刃",
+	"tetra/schematic/sword/rending_scissor_purple.slot1": "片剪太刀",
+
+	"tetra.module.sword/rending_scissor_purple.item_name": "片剪太刀",
+	"tetra.module.sword/rending_scissor_purple.name": "片剪太刀",
+	"tetra.module.sword/rending_scissor_purple.description": "用硬化的生命纤维制成的剪刀形剑刃",
+	"tetra.variant.rending_scissor_purple/rending_scissor_purple": "§5片剪太刀[紫]§r",
+
+	"tetra.advancements.upgrades.rending_scissor_red_upgrade.title": "保持内心坚定！",
+	"tetra.advancements.upgrades.rending_scissor_red_upgrade.description": "制作一把红色片剪太刀",

```

```suggestion
	"tetra.advancements.upgrades.rending_scissor_red_upgrade.description": "制作一柄红色片剪太刀",

```
（这里我也不知道把好还是柄好，但是大多都用的柄，我觉得统一比较好）